tensorflow/core/kernels/mkl_conv_grad_filter_ops.cc

```diff
@@ -266,8 +266,9 @@ class MklConv2DCustomBackpropFilterOp : public OpKernel {
     int input_offsets[2];
     size_t conv_strides[2];
     MklShape input_shape, grad_filter_shape, out_backprop_shape;
-    dnnPrimitive_t prim_conv_bwdfilter, convert_bwdfilter;
-    dnnLayout_t lt_input, lt_grad_filter, lt_out_backprop;
+    dnnPrimitive_t prim_conv_bwdfilter = nullptr, convert_bwdfilter = nullptr;
```

Comment:
break these out on to a line each:
dnnPrimitive_t prim_conv_bwdfilter = nullptr
dnnPrimitive_t convert_bwdfilter = nullptr

etc